### ISSUE TITLE
TypeError: Cannot read property 'start' of undefined

### ISSUE THREAD

**Issue by garthk** [launchdarkly/ld-vscode#17]
Also getting this every now and again. Haven't yet figured out the cause:
```
[2019-09-19 14:26:12.845] [exthost] [error] TypeError: Cannot read property 'start' of undefined
	at k (/Users/garth/.vscode/extensions/launchdarkly.launchdarkly-2.1.1/dist/extension.js:127:32319)
	at module.exports.t.LDFlagManager.LaunchDarklyCompletionItemProvider.provideCompletionItems (/Users/garth/.vscode/extensions/launchdarkly.launchdarkly-2.1.1/dist/extension.js:127:35483)
	at define.provideCompletionItems.o.asPromise (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:558:493)
```

**Comment by atrakh** [comment]
Hi @garthk ,

We've released a fix for this with [2.1.2](https://github.com/launchdarkly/ld-vscode/releases/tag/2.1.2). Please update the extension at your earliest convenience.

**Comment by groliveira18** [comment]
I just had the same problem after updating vscode. Has anyone managed to solve without returning the version of vscode?
